osi_environment.proto

```diff
@@ -324,4 +336,150 @@ message EnvironmentalConditions
         //
         optional uint32 seconds_since_midnight = 1;
     }
+
+    //
+    // \brief Description of a cloud layer.
+    //
+    message CloudLayer
+    {
+        // Properties of a cloud layer given by fractional cloud cover levels.
+        //
+        optional FractionalCloudCover fractional_cloud_cover = 1;
+
+        // Defines the fractional cloud cover [1] given by observation of total cloud amount in eights (oktas) of the sky.
+        //
+        // For visual alignment please see reference[2].
+        //
+        // The total degree of coverage indicates how much of the sky is covered with clouds altogether.
+        // 0 oktas means that there are no traces of clouds in the sky, \c
+        // 8 oktas means that the sky is completely covered with clouds and no sky blue can be recognized.
+        //
+        // \par References:
+        // [1] CIE engl. International Commission on Illumination. (2020). <em>CIE S017:2020 ILV: Intl. Lighitng Vocabulary, 2nd edn.</em>. Retrieved March 8, 2022, from https://cie.co.at/eilvterm/17-29-116
```

Comment:
A line break between reference [1] and [2] should be added. (I will do a final clean-up commit)